build/projects/Apps (W1)/.AL-Go/ImportTestDataInBcContainer.ps1

```diff
@@ -1,6 +0,0 @@
-Param(
```

Comment:
I think it would be better if we could keep this script (and therefore the demo data generation separate from the test execution). 

RunTestsInBcContainer.ps1 is run for every test app. ImportTestDataInBCContainer is only called once.

Reply:
Agree that we should keep demo data generation separate from the test execution.

---

I assume we have a sequence of execution like this:

```
ImportTestDataInBCContainer <== RunTestsInBcContainer
```

Thinking on top of my head, we probably need a bigger re-design if we'd like to run UnitTests on empty DB, but IntegrationTests with Demo Data?

Or maybe only run UnitTests in BCApps, and we can run IntegrationTests in NAV?